curriculum/challenges/english/25-front-end-development/quiz-javascript-math/66edc3ab8c6413c344f401bf.md

```diff
@@ -17,439 +17,452 @@ Answer all of the questions below correctly to pass the quiz.
+Which of the following statements are false about unary plus (+) operators?
 
 #### --distractors--
 
-Placeholder distractor 1
+It tries to convert the operand into a number
 
 ---
 
-Placeholder distractor 2
+It is the fastest way to convert an operand to a number
 
 ---
 
-Placeholder distractor 3
+It can convert operands with negative numbers except for hexadecimals
 
 #### --answer--
 
-Placeholder answer
+It can't convert `true`, `false`, `null` into a number
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which method in the Math standard built-in objects is used to round up a number?
 
 #### --distractors--
 
-Placeholder distractor 1
+`Math.round(num)`
 
 ---
 
-Placeholder distractor 2
+`Math.floor(num)`
 
 ---
 
-Placeholder distractor 3
+`Math.fround(num)`
 
 #### --answer--
 
-Placeholder answer
+`Math.ceil`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following Math.object methods are used to raise a base to a power?
 
 #### --distractors--
 
-Placeholder distractor 1
+`Math.exp(base, power)`
 
 ---
 
-Placeholder distractor 2
+`Math.raise(base, power)`
 
 ---
 
-Placeholder distractor 3
+`Math.e(base, power)`
 
 #### --answer--
 
-Placeholder answer
+`Math.pow(base, power)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct method for generating a random number in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+`Math.rand(1,10)`
 
 ---
 
-Placeholder distractor 2
+`Math.rand()`
 
 ---
 
-Placeholder distractor 3
+`Math.random(1,10)`
 
 #### --answer--
 
-Placeholder answer
+`Math.random()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will return false when using isNaN()?
 
 #### --distractors--
 
-Placeholder distractor 1
+`isNaN("1998", "1999")`
 
 ---
 
-Placeholder distractor 2
+`isNaN({})`
 
 ---
 
-Placeholder distractor 3
+`isNaN(undefined)`
 
 #### --answer--
 
-Placeholder answer
+`isNaN(null)`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will convert 172 to its hexadecimal equivalent?
 
 #### --distractors--
 
-Placeholder distractor 1
+`console.log((-0xa5).toString(2));`
 
 ---
 
-Placeholder distractor 2
+`console.log((172).string(0x));`
 
 ---
 
-Placeholder distractor 3
+`console.log((172).parseString(0x));`
 
 #### --answer--
 
-Placeholder answer
+`console.log((172).toString(16));`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the right way to convert string to integer?
 
 #### --distractors--
 
-Placeholder distractor 1
+`console.log(parseINT("300"))`
 
 ---
 
-Placeholder distractor 2
+`console.log(int("300"))`
 
 ---
 
-Placeholder distractor 3
+`console.log(integer("300"))`
 
 #### --answer--
 
-Placeholder answer
+`console.log(parseInt("300"))`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statement is false about parseFloat()?
```

Comment:
```suggestion
Which of the following statement is false about `parseFloat()`?
```

code inside a sentence also needs backticks, please fix the others

Reply:
Hi ilenia, i've made the changes